docker-compose.yml

```diff
@@ -111,6 +111,35 @@ services:
     networks:
       - pmm-network
 
+  # it needs for deleting postgres db for test T1087
+  postgresremovedefaultdb:
```

Comment:
@ElenaFedichkina could you not have a separate compose file for this specific case, you could always start the containers, in your test and kill them post that.